packages/cli/tests/lib/cli.js

```diff
@@ -7,10 +7,12 @@ const shell = require('shelljs');
 const root = join(__dirname, '../../../..');
 
 async function linkPackage(name, from, to) {
-	await symlink(
-		join(from, 'node_modules', name),
-		join(to, 'node_modules', name)
-	);
+	try {
+		await symlink(
+			join(from, 'node_modules', name),
+			join(to, 'node_modules', name)
+		);
+	} catch {}
 }
```

Comment:
In case this look odd: when running multiple builds in the same test fixture (such as testing `option: true` & `option: false` together), symlinking would be triggered twice, as it's called on build. `fs.symlink()` throws if the symlink already exists, so we should catch & ignore those cases.